modules/oadp-usecase-kopia-override-algorithms.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc          
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-usecase-kopia-override-algorithms.adoc_{context}"]
+= Use case for overriding Kopia hashing, encryption, and splitter algorithms
+
+This use case walks you through the steps for taking a backup with Kopia environment variables for hashing, encryption, and splitter. You store the backup in an AWS S3 bucket. You then verify the environment variables by connecting to the Kopia repository.
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have an AWS S3 bucket configured as the backup storage location.
+* You have created the secret by using the credentials provided by the cloud provider.
+* You have installed the Kopia client.
```

Comment:
Not required for using it, required for verification part.

Reply:
I am keeping this one as we have a 'Verification' section at the end and user runs the kopia command to verify.